.env.sample

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  from: {
+    HOST: 'HOST',
+    USER: 'USER',
+    PASS: 'PASS',
+    DB: 'DB_FROM',
+    TABLE: 'TABLE_NAME_OPTIONAL',
+    VIEW: 'VIEW_NAME_OPTIONAL',
+  },
+  target: {
+    HOST: 'HOST',
+    USER: 'USER',
+    PASS: 'PASS',
+    DB: 'TESTE',
+  }
```

Comment:
There is no need for a ```target``` here. Lets keep it simple and not generate the knexfile.js.